web/src/engine/websites/Manjanoon.ts

```diff
@@ -1,20 +1,26 @@
 import { Tags } from '../Tags';
 import icon from './Manjanoon.webp';
-import { DecoratableMangaScraper } from '../providers/MangaPlugin';
-import * as MangaStream from './decorators/WordPressMangaStream';
+import { type Chapter, DecoratableMangaScraper, Page } from '../providers/MangaPlugin';
 import * as Common from './decorators/Common';
+import { FetchHTML } from '../platform/FetchProvider';
 
-@MangaStream.MangaCSS(/^{origin}\/manga\/[^/]+\/$/)
-@MangaStream.MangasSinglePageCSS()
-@MangaStream.ChaptersSinglePageCSS()
-@MangaStream.PagesSinglePageJS()
+@Common.MangaCSS(/^{origin}\/series\/[^/]+$/, 'div.grid div.grid h1')
+@Common.MangasSinglePagesCSS(['/latest'], 'div.grid a.grid', Common.AnchorInfoExtractor(true))
+@Common.ChaptersSinglePageCSS('div#chapters a', Common.AnchorInfoExtractor(true))
 @Common.ImageAjax()
 export default class extends DecoratableMangaScraper {
 
     public constructor() {
-        super('manjanoon', 'Manjanoon', 'https://vrnoin.site', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.Arabic, Tags.Source.Scanlator);
+        super('manjanoon', 'Manjanoon', 'https://manjanoon.art', Tags.Media.Manhwa, Tags.Media.Manhua, Tags.Media.Manga, Tags.Language.Arabic, Tags.Source.Scanlator);
     }
     public override get Icon() {
         return icon;
     }
+
+    public override async FetchPages(chapter: Chapter): Promise<Page[]> {
```

Comment:
May use `PagesSinglePageJS` instead

Reply:
Irk, but i'd rather refrain opening a browser window and executing foreign code when i can.
Plus, doing this way is lighter i guess.